<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolAddQuad">
        <property name="text">
         <string>+Quad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolAddPoly">
        <property name="text">
         <string>+Poly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolAddTileset">
        <property name="text">
         <string>+Tileset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolAddStretchPatch">
        <property name="text">
         <string>+Patch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolAddQuadArray">
        <property name="text">
         <string>+QuadArray</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolRemove">
        <property name="text">
         <string>REMOVE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolClone">
        <property name="text">
         <string>CLONE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="listPanel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="containerSpriteTree" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="settingsContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="previewPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="canvasContainer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="toolContainer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageQuad">
           <widget class="QWidget" name="originPanel" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>81</width>
              <height>81</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolOriginNW">
               <property name="text">
                <string>┌</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="toolOriginN">
               <property name="text">
                <string>─</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QToolButton" name="toolOriginNE">
               <property name="text">
                <string>┐</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QToolButton" name="toolOriginW">
               <property name="text">
                <string>│</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="toolOriginC">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QToolButton" name="toolOriginE">
               <property name="text">
                <string>│</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QToolButton" name="toolOriginSW">
               <property name="text">
                <string>└</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QToolButton" name="toolOriginSE">
               <property name="text">
                <string>┘</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QToolButton" name="toolOriginS">
               <property name="text">
                <string>─</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QCheckBox" name="checkAlphaView">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>20</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>ALPHA VIEW</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
